src/overlays/actors/ovl_En_Aob_01/z_en_aob_01.c

```diff
@@ -42,92 +56,1011 @@ const ActorInit En_Aob_01_InitVars = {
     (ActorFunc)EnAob01_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_809C3820 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK1, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ActorAnimationEntry D_809C3790[6] = {
+    { &D_06007758, 1.0f, 0.0f, 0.0f, 0, 0.0f }, { &D_060068B4, 1.0f, 0.0f, 0.0f, 2, 0.0f },
+    { &D_0600700C, 1.0f, 0.0f, 0.0f, 0, 0.0f }, { &D_060058EC, 1.0f, 0.0f, 0.0f, 2, 0.0f },
+    { &D_06006040, 1.0f, 0.0f, 0.0f, 0, 0.0f }, { &D_06007758, 1.0f, 0.0f, 0.0f, 0, -6.0f },
+};
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 18, 64, 0, { 0, 0, 0 } },
 };
 
-#endif
+typedef struct {
+    Vec3f unk_00;
+    s16 unk_04;
+    s16 unk_06;
+} EnAobStruct;
+
+EnAobStruct sAnimations[] = {
+    { { -4130.0f, 150.0f, 1367.0f }, 84, 0 },  { { -4861.0f, 172.0f, 1606.0f }, 94, 4 },
+    { { -4139.0f, 155.0f, 2133.0f }, 73, 6 },  { { -4406.0f, 144.0f, 1416.0f }, 88, 2 },
+    { { -4156.0f, 155.0f, 1731.0f }, 42, 0 },  { { -4033.0f, 157.0f, 1994.0f }, -65, 1 },
+    { { -4582.0f, 158.0f, 1206.0f }, 144, 2 }, { { -4595.0f, 156.0f, 1493.0f }, 61, 3 },
+    { { -4526.0f, 146.0f, 1702.0f }, 61, 4 },  { { -3820.0f, 162.0f, 1965.0f }, 109, 5 },
+    { { -4395.0f, 147.0f, 1569.0f }, -24, 0 }, { { -4315.0f, 150.0f, 2048.0f }, 61, 6 },
+    { { -4827.0f, 168.0f, 1328.0f }, 115, 4 }, { { -4130.0f, 150.0f, 1367.0f }, 112, 0 },
+};
```

Comment:
These have nothing to do with animations. These are related to actor spawn properties such as position, rotation, and params